docs/reference/setup.asciidoc

```diff
@@ -67,13 +69,13 @@ include::settings/monitoring-settings.asciidoc[]
 
 include::modules/indices/query_cache.asciidoc[]
 
-include::modules/indices/request_cache.asciidoc[]
-
 include::modules/indices/search-settings.asciidoc[]
 
+include::modules/indices/request_cache.asciidoc[]
```

Comment:
When rendered this title is `Shard request cache settings`. We may want to move it after `Security settings`.